test/e2e/storage/testsuites/subpath.go

```diff
@@ -1026,7 +1026,7 @@ func testSubpathReconstruction(f *framework.Framework, hostExec utils.HostExec,
 	pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.TODO(), pod, metav1.CreateOptions{})
 	framework.ExpectNoError(err, "while creating pod")
 
-	err = e2epod.WaitForPodRunningInNamespace(f.ClientSet, pod)
+	err = framework.WaitForPodRunningInNamespaceWithOS(f.ClientSet, pod)
```

Comment:
Maybe can this be named `WaitForPodRunningInNamespaceWithPerOSTimeouts`?
I like the intentions of these changes but think the function name could be more descriptive.